lib/LaTeXML/Package/longtable.sty.ltxml

```diff
@@ -24,10 +24,15 @@ DefMacro('\longtable[]{}',
   '\@longtable@bindings{#2}\@@longtable[#1]{#2}\@start@alignment');
 DefMacro('\endlongtable',
   '\@finish@alignment\@end@tabular');
-DefMacro('\csname longtable*\endcsname []{}',
-  '\@longtable@bindings{#2}\@@longtable[#1]{#2}\@start@alignment');
-DefMacro('\csname endlongtable*\endcsname',
-  '\@finish@alignment\@end@tabular');
+# {longtable*} is not defined in longtable.sty
+# and if we're trying to mimic {tabular*},
+# then this needs \endcsname{Dimension}[]{}
+#DefMacro('\csname longtable*\endcsname []{}',
+#  '\@longtable@bindings{#2}\@@longtable[#1]{#2}\@start@alignment');
+#DefMacro('\csname endlongtable*\endcsname',
+#  '\@finish@alignment\@end@tabular');
```

Comment:
Please don't remove `longtable*`, maybe just enhance it?

In texlive-proper, the variety of versioned revtex classes (such as `revtex4-1.cls`) defined `longtable*`, and in latexml that is realized via including the longtable package and adding the star variant in it.

Maybe moving the definition of `longtable*` to `revtex4_support.sty.ltxml` is "more correct" in respecting the texlive organization, but I don't see any harm with leaving it in longtable.sty.ltxml for now.

Reply:
I missed that it was defined in revtex4-1.  longtable* is not mimicking tabular*; instead, longtable* spans the whole page in a two column document.  So changing that can wait until we figure out how to deal with columns.